CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+Contributing
+============
+
+## Prerequisites
+To release the SDK from your local development device you need:
+- Node installed on your development device
+- Clone https://github.com/bitvavo/node-bitvavo-api to your development device
+- The [Bitvavo password for NPM] from the Exchange Backbone team vault in 1Password
+
+## Test and Publish the SDK
+After you have made your changes to the SDK, to publish Bitvavo SDK for Node.js:
+1. **Test the package**
+   1. Run the smoke tests in `<repo-root>/example/testApi.js`
+2. **Release the SDK**
+   1. Please check the current version and decide which new version to use.
```

Comment:
Good stuff, nearly there. Best practice would be to trigger the action when the user creates a new release. I suggest:

2. **Release the SDK**
    In this GitHub repo, [Draft a new release]. Use [Semver] to choose the version number. 

The new version is published to https://www.npmjs.com/package/bitvavo. 


[Draft a new release]: https://github.com/bitvavo/node-bitvavo-api/releases
[Semver]: https://semver.org/


Tech writerly things: 
- Never show doubt - people will ask what happens on an unsuccessful fun and then we go in to existentialist philosophy. 
- Never say please - it is easy to trip yourself up in politeness, plus, it adds loads of words nobody wants to read. 
- Always use the present tense - check the Google Developer Style Guide.